Autopilot/Src/main.c

```diff
@@ -129,9 +130,20 @@ int main(void)
   /* USER CODE BEGIN 2 */
   debug("\n\nStarting up...");
   debug("Compiled on %s at %s", __DATE__, __TIME__);
-  
+
   HAL_GPIO_WritePin(LED1_GPIO_Port, LED1_Pin, GPIO_PIN_SET);
 
+  EEPROM_Init();
+
+  Interchip_StoA_Packet* dataRX = malloc(sizeof(Interchip_StoA_Packet));
+  Interchip_AtoS_Packet* dataTX = malloc(sizeof(Interchip_AtoS_Packet));
+  interchipInit(dataTX, dataRX);
+
+  while(1){
+
+    interchipUpdate();
+    HAL_Delay(100);
+  }
```

Comment:
If possible, I'd like to see this running from an OS thread.

Reply:
I was going to leave that for a future PR, just wanted to get the basic driver working.

Reply:
Valid. Fine as is.